BE/src/main/java/louie/dong/airbnb/accommodation/dto/AccommodationDetailResponse.java

```diff
@@ -34,6 +37,14 @@ public AccommodationDetailResponse(Accommodation accommodation, boolean wish) {
 		this.roomInformation = accommodation.getRoomInformation();
 		this.description = accommodation.getDescription();
 		this.price = accommodation.getPrice();
-		this.wishlist = wish;
+		this.isWish = wish;
+	}
+
+	private String getImageUrl(List<AccommodationImage> accommodationImages) {
+		try {
+			return accommodationImages.get(0).getImageUrl();
+		} catch (NoSuchElementException e) {
+			throw new NoSuchElementException("숙소 이미지가 존재하지 않습니다.");
+		}
```

Comment:
제가 설명이 부족했네요 ㅠ
숙소 이미지가 존재하지 않는게 예외상황이라면 예외를 발생시켜도 좋지만,
제가 의도한 바는 예외 발생 없이 정상동작하는거였습니다.

이미지가 없으면 **여기** 에서 예외가 발생할겁니다.
accommodationImages.get(0)**여기**.getImageUrl()

Accommodation에 메소드를 생성한다 치고 예시를 들어드릴게요
```suggestion
class Accommodation{
// ..

	private String getThumbnail() {
		if(hasImage()){
			// 첫번째 이미지 리턴
		}
		return null;
	}
```